src/routes/safe/components/Transactions/TxList/TxInfoCreation.tsx

```diff
@@ -21,7 +26,7 @@ export const TxInfoCreation = ({ transaction }: { transaction: Transaction }): R
   return (
     <TxDetailsContainer>
       <div className="tx-summary">
-        <div className="tx-hash">
+        <StyledGridRow>
```

Comment:
It seems like every `<StyledGridRow>` contains a `<Text>` 'title'. You could move that into the component as pass it as a prop. The `children` could then be the 'value'.

Reply:
The `StyledGridRow` is just a styled div. Is it worth making a component out of it?